SWLOR.Game.Server/Feature/AbilityDefinition/FirstAid/MedKitAbilityDefinition.cs

```diff
@@ -48,6 +51,13 @@ private void Impact(uint activator, uint target, int baseAmount)
             var willpowerMod = GetAbilityModifier(AbilityType.Willpower, activator);
             var amount = baseAmount + willpowerMod * 10 + Random.D10(1);
 
+            // limit xp to amount *actually* healed.
+            if (amount > (GetMaxHitPoints(target) - GetCurrentHitPoints(target)))
+            {
+                amount = GetMaxHitPoints(target) - GetCurrentHitPoints(target);
+            }
+
+            Service.Skill.GiveSkillXP(activator, SkillType.FirstAid, 3 * amount);
```

Comment:
Nitpick: When possible, default to using statements and remove the namespacing within the code. In this example, removing Service.Skill.GiveSkillXP and replacing it with just Skill.GiveSkillXP.

If there's overlap in class names it's fine to prefix with Service here but I think you can get away without it in this case.

Reply:
Skill conflicted with something else so I needed to qualify it - I forget what.  But I think it was an enum called Skill (maybe in the base NWN stuff, using Skill rather than SkillType?).

Reply:
I think I know what you're talking about. I believe Core has a Skill class too. I've run into that before. All good then, feel free to leave it.